helm/botkube/templates/clusterrolebinding.yaml

```diff
@@ -13,6 +13,24 @@ roleRef:
   kind: ClusterRole
   name: {{ include "botkube.fullname" . }}-clusterrole
 subjects:
+- kind: ServiceAccount
+  name: {{ include "botkube.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: botkube-plugins-read-only
+  labels:
+    app.kubernetes.io/name: {{ include "botkube.name" . }}
+    helm.sh/chart: {{ include "botkube.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: botkube-plugins-read-only
+subjects:
 - kind: ServiceAccount
   name: {{ include "botkube.serviceAccountName" . }}
   namespace: {{ .Release.Namespace }}
```

Comment:
Also, maybe this should be configurable too, just to make it visibile for users that this subject is used in `rbac.context` for plugins 🤔 WDYT?

Reply:
I see more issues than benefits here. The idea of this ClusterRoleBinding is to retain current functionality.
I think that once we progress with implementing RBAC, there will be more specific configuration options for users to use.

Reply:
Sorry, I meant just the group name, not the whole subject. The only problem is that users will need to use the name in the config so they do need to know it 🤔 How they can learn it intuitively, if it is not found in the `values.yaml` doesn't support it? 🤔 I think that wouldn't cause too much issues, as by default we will keep it as `botkube-plugins-read-only` and in that way user will be able to find the occurrence when configuring Helm chart.